packages/jira-adapter/src/filters/layouts/request_types_layouts_to_values.ts

```diff
@@ -0,0 +1,52 @@
+/*
+*                      Copyright 2023 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import { Field, isInstanceElement, isObjectType } from '@salto-io/adapter-api'
+import _ from 'lodash'
+import { FilterCreator } from '../../filter'
+import { ISSUE_VIEW_TYPE, REQUEST_FORM_TYPE, REQUEST_TYPE_NAME } from '../../constants'
+
+const supportedTypes = [REQUEST_FORM_TYPE, ISSUE_VIEW_TYPE]
```

Comment:
nit: this is a constant, please use uppercase
```suggestion
const SUPPORTED_TYPES = [REQUEST_FORM_TYPE, ISSUE_VIEW_TYPE]
```
also - not sure what "supported" means here, these are just the types that need to be adjusted manually, right? maybe something like `LAYOUT_TYPES_TO_ADJUST` (or anything else that explains what this is)?